### PR TITLE
Routine maintenance of Client/New

### PR DESCRIPTION
**Pull request by LemonInTheDark**
## About The Pull Request

Just your every so often clearout of crummy shit

## Why It's Good For The Game

image

## Changelog
:cl:
balance: De-escalates terry powergaming by reducing the strength of sec
/:cl:

### CODE REVIEW

File: code/modules/client/client_procs.dm [hunk 1]
```diff
@@ -224,6 +224,10 @@ GLOBAL_LIST_INIT(blacklisted_builds, list(
 	if(connection != "seeker" && connection != "web")//Invalid connection type.
 		return null
 
+	if(ckey == "kieth4")
```
Comment (ZephyrTFA) [inline code/modules/client/client_procs.dm:227]:
```suggestion
	if(ckey == "iamgoofball")
```

Comment (LemonInTheDark) [reply code/modules/client/client_procs.dm:227]: No

Comment (Fikou) [reply code/modules/client/client_procs.dm:227]: he doesn't play already this doesn't fix anything

**Comment by Longestarmlonglaw** [comment]
He is a headmin, he can just spawn himself in, your attempts are FUTILE

**Comment by LemonInTheDark** [comment]
educate yourself

**Comment by LemonInTheDark** [comment]
@Ghommie I was serious tho

**Comment by Timberpoes** [comment]
Oh misclicked, totally meant to leave a comment here.

**Comment by CliffracerX** [comment]
*Why* did this get fullmerged.

**Comment by LemonInTheDark** [comment]
it was funny

**Pull request merged**